src/core/server/elasticsearch/version_check/ensure_es_version.ts

```diff
@@ -121,7 +138,8 @@ function compareNodes(prev: NodesVersionCompatibility, curr: NodesVersionCompati
     curr.incompatibleNodes.length === prev.incompatibleNodes.length &&
     curr.warningNodes.length === prev.warningNodes.length &&
     curr.incompatibleNodes.every((node, i) => nodesEqual(node, prev.incompatibleNodes[i])) &&
-    curr.warningNodes.every((node, i) => nodesEqual(node, prev.warningNodes[i]))
+    curr.warningNodes.every((node, i) => nodesEqual(node, prev.warningNodes[i])) &&
+    compareNodesInfoErrorMessages(curr, prev)
```

Comment:
can you explain why `curr.message === prev.message` is not enough? 

| Scenario | `prev.message` | `curr.message` | `curr.message === prev.message` |
|----------|----------------|----------------|:-------------------------------:|
| No errors | `undefined` | `undefined` | `true` |
| New error happens | `undefined` | `Error 1` | `false` |
| Same error happens | `Error 1` | `Error 1` | `true` |
| Different error happens | `Error 1` | `Error 2` | `false` |
| Back to normal | `Error 2` | `undefined` | `true` |

Reply:
The message fields [aren't `undefined`](https://github.com/TinaHeiligers/kibana/pull/57/files/a432b99fa111b2afdf7fe6c6059d2eeb1ffdf0b5#diff-c7e7076ab5d8fbeddfe2d1bd7a5b90578b09a6638ef0180d2cb75e6751234385L95) when there isn't an error. We might get the same message for different situations.